var/spack/repos/builtin/packages/python/package.py

```diff
@@ -415,19 +416,26 @@ class Python(Package):
     # C/C++ modules, consider installing a Spack-managed Python with
     # this patch instead. For more information, see:
     # https://github.com/spack/spack/pull/16856
+    #
+    # NOTE: distutils was deprecated in 3.10, so we don't add new patches
```

Comment:
This probably requires an update to py-numpy and py-scipy, if not others.

Reply:
Yes, all versions of numpy still use distutils. We can't just ignore these patches.

Reply:
py-numpy builds fine for me, py-scipy already has an upperbound of the form depends_on(python@:3.10)

Reply:
okay you win i added the patch back :|